Makefile

```diff
@@ -15,6 +15,36 @@ ut: update-version
 	calico/test \
 	nosetests tests/unit -c nose.cfg
 
+# Run system tests.
+st: docker-image run-etcd run-k8s-apiserver
```

Comment:
Actually no, it should work fine for me to do this now, and I think #86 will still merge straightforwardly with it.